src/pages/Events/EventDoorCounter.tsx

```
@@ -0,0 +1,210 @@
+  { id: 'evt-1', eventName: 'AI & ML Hackathon 2026', venue: 'Innovation Hub', capacity: 250, attendees: 198 },
+  { id: 'evt-2', eventName: 'Global Cultural Night', venue: 'Campus Green Pavilion', capacity: 500, attendees: 312 },
+  { id: 'evt-3', eventName: 'Quantum Physics Symposium', venue: 'Science Complex', capacity: 120, attendees: 75 },
+];
+
+export default function EventDoorCounter() {
+  const [events, setEvents] = useState<EventData[]>(INITIAL_EVENTS);
+  const [selectedEventId, setSelectedEventId] = useState<string>(INITIAL_EVENTS[0].id);
+  const [doorCount, setDoorCount] = useState<number>(0);
+  const [isRealtime, setIsRealtime] = useState<boolean>(true);
+
+  const selectedEvent = events.find(event => event.id === selectedEventId) || INITIAL_EVENTS[0];
```

ESLint

[error] Replace `event` with `(event)`

(prettier/prettier)